DOCS-683 - New in Copilot - Dynamic Titles, Alert Troubleshooting, Pinned Suggestions (#5067)
SumoLogic/sumologic-documentation: docs/search/copilot.md

### Step 4: Open in Log Search

> You can open your Copilot query in [Log Search](/docs/search) to access Sumo Logic’s full search functionality, allowing you to investigate further, refine your query, save the search, and take action as needed.

Author: ```suggestion
You can open your Copilot query in [Log Search](/docs/search) to access Sumo Logic’s full search functionality. This allows you to continue investigating, refine your query, save the search, or take action as needed.
```

### Step 4: Open in Log Search

> You can open your Copilot query in [Log Search](/docs/search) to access Sumo Logic’s full search functionality. This allows you to continue investigating, refine your query, save the search, or take action as needed.

Author: ```suggestion
You can open your query in [Log Search](/docs/search) to access Sumo Logic’s full search functionality. This allows you to continue investigating, refine your query, save the search, or take action as needed.
```
